pygmt/src/blockm.py

```diff
@@ -16,10 +17,10 @@
 
 def _blockm(block_method, data, x, y, z, outfile, **kwargs):
     r"""
-    Block average (x,y,z) data tables by mean, median, or mode estimation.
+    Block average (x, y, z) data tables by mean, median, or mode estimation.
 
-    Reads arbitrarily located (x,y,z) triples [or optionally weighted
-    quadruples (x,y,z,w)] from a table and writes to the output a mean,
+    Reads arbitrarily located (x, y, z) triplets [or optionally weighted
```

Comment:
I'm not sure about the changes from "triples" to "triplets". The GMT upstream documentation use "triples".

We need reviews from a native English speaker (maybe @maxrjones).

Reply:
Thanks @seisman for your review and comment! I was/am also unsure about the term "triples" or "triplets". 

Actually, the GMT documentation is also not consistent regarding this term, e.g. [blockmean](https://docs.generic-mapping-tools.org/dev/blockmean.html?highlight=triples) uses "triples", but [wiggle](https://docs.generic-mapping-tools.org/dev/wiggle.html?highlight=triplets) uses "triplets". Thus, I agree, a review from a native English speaker is needed. In case I changed it in the wrong direction, I am of course pleased to change this the other way around :blush:.

----------------------
**GMT**
```
/cygdrive/c/users/Admin/C2/EigeneDokumente/Studium/Promotion/01_GitHub/github_gmt/gmt/doc/rst/source
$ grep 'triplets' *.rst
blockmedian.rst:and output a binary table with double precision triplets, run::
blockmode.rst:and output a binary table with double precision triplets, run::
contour.rst:    triplets of node numbers for a triangle [Default computes these
contour.rst:.. |Add_-bi| replace:: [Default is 3 input columns]. Use 4-byte integer triplets for node ids (|-E|).
gmtmath.rst:| **HSV2LAB**     | 3 3    | Convert h,s,v triplets to l,a,b triplets, with h = A (0-360), s = B and v = C (0-1)        |
gmtmath.rst:| **HSV2RGB**     | 3 3    | Convert h,s,v triplets to r,g,b triplets, with h = A (0-360), s = B and v = C (0-1)        |
gmtmath.rst:| **HSV2XYZ**     | 3 3    | Convert h,s,v triplets to x,t,z triplets, with h = A (0-360), s = B and v = C (0-1)        |
gmtmath.rst:| **LAB2HSV**     | 3 3    | Convert l,a,b triplets to h,s,v triplets                                                   |
gmtmath.rst:| **LAB2RGB**     | 3 3    | Convert l,a,b triplets to r,g,b triplets                                                   |
gmtmath.rst:| **LAB2XYZ**     | 3 3    | Convert l,a,b triplets to x,y,z triplets                                                   |
gmtmath.rst:| **RGB2HSV**     | 3 3    | Convert r,g,b triplets to h,s,v triplets, with r = A, g = B, and b = C (in 0-255 range)    |
gmtmath.rst:| **RGB2LAB**     | 3 3    | Convert r,g,b triplets to l,a,b triplets, with r = A, g = B, and b = C (in 0-255 range)    |
gmtmath.rst:| **RGB2XYZ**     | 3 3    | Convert r,g,b triplets to x,y,x triplets, with r = A, g = B, and b = C (in 0-255 range)    |
gmtmath.rst:| **XYZ2HSV**     | 3 3    | Convert x,y,z triplets to h,s,v triplets                                                   |
gmtmath.rst:| **XYZ2LAB**     | 3 3    | Convert x,y,z triplets to l,a,b triplets                                                   |
gmtmath.rst:| **XYZ2RGB**     | 3 3    | Convert x,y,z triplets to r,g,b triplets                                                   |
grd2xyz.rst:xyz-triplets in ASCII [or binary] format to standard output. Modify the
grd2xyz.rst:to dump the original *x y z* triplets and then reprocess the data onto an equidistant
grdmath.rst:| **HSV2RGB**   | 3 3   | Convert h,s,v triplets to r,g,b triplets, with h = A (0-360), s = B and v = C (both in 0-1)            |
grdmath.rst:| **HSV2RGB**   | 3 3   | Convert h,s,v triplets to r,g,b triplets, with h = A (0-360), s = B and v = C (0-1)                    |
grdmath.rst:| **HSV2XYZ**   | 3 3   | Convert h,s,v triplets to x,t,z triplets, with h = A (0-360), s = B and v = C (0-1)                    |
grdmath.rst:| **LAB2HSV**   | 3 3   | Convert l,a,b triplets to h,s,v triplets                                                               |
grdmath.rst:| **LAB2RGB**   | 3 3   | Convert l,a,b triplets to r,g,b triplets                                                               |
grdmath.rst:| **LAB2XYZ**   | 3 3   | Convert l,a,b triplets to x,y,z triplets                                                               |
grdmath.rst:| **RGB2HSV**   | 3 3   | Convert r,g,b triplets to h,s,v triplets, with r = A, g = B, and b = C (all in 0-255 range)            |
grdmath.rst:| **RGB2LAB**   | 3 3   | Convert r,g,b triplets to l,a,b triplets, with r = A, g = B, and b = C (in 0-255 range)                |
grdmath.rst:| **RGB2XYZ**   | 3 3   | Convert r,g,b triplets to x,y,x triplets, with r = A, g = B, and b = C (in 0-255 range)                |
grdmath.rst:| **XYZ2HSV**   | 3 3   | Convert x,y,x triplets to h,s,v triplets                                                               |
grdmath.rst:| **XYZ2LAB**   | 3 3   | Convert x,y,x triplets to l,a,b triplets                                                               |
grdmath.rst:| **XYZ2RGB**   | 3 3   | Convert x,y,x triplets to r,g,b triplets                                                               |
plot3d.rst:Reads (x,y,z) triplets from *files* [or standard input] and
triangulate.rst:default, the output is triplets of point id numbers that make up each
triangulate.rst:    triplets of node numbers for a triangle in the input *table* [Default computes these
triangulate.rst:    Used in conjunction with |-G| to also write the triplets of the
triangulate.rst:.. |Add_-bo| replace:: [Default is same as input].  Node ids are stored as double triplets.
wiggle.rst:Reads (*x*,\ *y*,\ *z*) triplets from files [or standard
xyz2grd.rst:    files holding *z* or *x, y, z* values. The *x, y, z* triplets do not have to
```
```
/cygdrive/c/users/Admin/C2/EigeneDokumente/Studium/Promotion/01_GitHub/github_gmt/gmt/doc/rst/source
$ grep 'triples' *.rst
blockmean.rst:**blockmean** reads arbitrarily located (*x*,\ *y*,\ *z*) triples [or
blockmedian.rst:**blockmedian** reads arbitrarily located (*x*,\ *y*,\ *z*) triples [or
blockmode.rst:**blockmode** reads arbitrarily located (*x*,\ *y*,\ *z*) triples [or
nearneighbor.rst:**nearneighbor** reads arbitrarily located (*x,y,z*\ [,\ *w*]) triples
project.rst:triples at equal increments *dist* along a profile using |-G|. In this case, no input is read.
surface.rst:**surface** reads randomly-spaced (x,y,z) triples from standard input
```

**PyGMT**
```
/cygdrive/c/users/Admin/C2/EigeneDokumente/Studium/Promotion/01_GitHub/github_pygmt/pygmt/pygmt/src
$ grep 'triplets' *.py
blockm.py:    Takes a matrix, xyz triplets, or a file name as input.
blockm.py:    Takes a matrix, xyz triplets, or a file name as input.
blockm.py:    Takes a matrix, xyz triplets, or a file name as input.
contour.py:    Takes a matrix, (x,y,z) triplets, or a file name as input and plots lines,
grd2xyz.py:    Read a grid and output xyz-triplets as a :class:`numpy.ndarray`,
plot3d.py:    Takes a matrix, (x,y,z) triplets, or a file name as input and plots
triangulate.py:    is calculated. By default, the output is triplets of point id numbers that
wiggle.py:    Takes a matrix, (x,y,z) triplets, or a file name as input and plots z as a
```
```
 /cygdrive/c/users/Admin/C2/EigeneDokumente/Studium/Promotion/01_GitHub/github_pygmt/pygmt/pygmt/src
$ grep 'triples' *.py
blockm.py:    Reads arbitrarily located (x,y,z) triples [or optionally weighted
blockm.py:    Reads arbitrarily located (x,y,z) triples [or optionally weighted
blockm.py:    Reads arbitrarily located (x,y,z) triples [or optionally weighted
blockm.py:    Reads arbitrarily located (x,y,z) triples [or optionally weighted
nearneighbor.py:    **nearneighbor** reads arbitrarily located (*x,y,z*\ [,\ *w*]) triples
nearneighbor.py:    Takes a matrix, xyz triples, or a file name as input.
project.py:    :math:`(r, s, p)` triples at equal increments along a profile using the
surface.py:    Surface reads randomly-spaced (x,y,z) triples and produces gridded values
surface.py:    Takes a matrix, xyz triples, or a file name as input.
triangulate.py:    def delaunay_triples(  # pylint: disable=too-many-arguments,too-many-locals
```

Reply:
triplets works well to describe sets of three

Reply:
Great! Thanks @maxrjones for your response. 
So, we leave this PyGMT PR as it is and I try to submit a corresponding PR for the upstream GMT documentation :wink:.